detections/endpoint/linux_auditd_add_user_account.yml

```diff
@@ -0,0 +1,63 @@
+name: Linux Auditd Add User Account
+id: aae66dc0-74b4-4807-b480-b35f8027abb4
+version: 1
+date: '2024-08-27'
+author: Teoderick Contreras, Splunk
+status: production
+type: Anomaly
+description: The following analytic detects the creation of new user accounts on Linux
+  systems using commands like "useradd" or "adduser." It leverages data from Endpoint
+  Detection and Response (EDR) agents, focusing on process names and command-line
+  executions. This activity is significant as adversaries often create new user accounts
+  to establish persistence on compromised hosts. If confirmed malicious, this could
+  allow attackers to maintain access, escalate privileges, and further compromise
+  the system, posing a severe security risk.
+data_source:
+- Linux Auditd Proctitle
+search: '`linux_auditd` `linux_auditd_normalized_proctitle_process`| rename host as dest | where LIKE (process_exec,
+  "%useradd%") OR LIKE (process_exec, "%adduser%") | stats count min(_time) as firstTime
+  max(_time) as lastTime by process_exec proctitle dest  | `security_content_ctime(firstTime)`|
+  `security_content_ctime(lastTime)`| `linux_auditd_add_user_account_filter`'
+how_to_implement: The detection is based on data that originates from Endpoint Detection
```

Comment:
@tccontre : can we update the how to implement to reflect details on data ingestion from `linux_auditd` and which TA does it and other relevant information